scripts/externalTests/common.sh

```diff
@@ -106,25 +106,31 @@ function setup_solc
 
 function download_project
 {
-    local repo="$1"
+    local repo_url="$1"
     local ref_type="$2"
-    local solcjs_ref="$3"
+    local ref="$3"
     local test_dir="$4"
 
-    [[ $ref_type == commit || $ref_type == branch || $ref_type == tag ]] || assertFail
+    [[ $ref_type == commit || $ref_type == branch || $ref_type == tag || $ref_type == release ]] || assertFail
```

Comment:
Now that I'm thinking again about it, we could actually remove the `ref_type` completely.

Reply:
I remove it in a separated commit, in case you think it is a good idea.

Reply:
It's fine. The reason we had this parameter was that `git clone --branch <ref>` won't handle commits so I had to distinguish them and handle commits separately. If it's now all handled uniformly by the new implementation, we no longer have to know that so we can simplify things.